packages/docusaurus-plugin-content-docs/src/cli.ts

```diff
@@ -93,7 +93,10 @@ export function cliDocsVersionCommand(
   }
 
   // Load current sidebar and create a new versioned sidebars file.
-  if (fs.existsSync(sidebarPath)) {
+  if (
```

Comment:
would be simpler to handle the false value in loadSidebars

Reply:
I don't feel the existsSync call is really useful btw, if the user provided a bad path an error is already displayed when he starts his site